Automattic/jetpack
# Jetstream / Launchpad: initial skeleton for the plugin

## Proposed changes:

* Bring in the structure for a simple plugin that does nothing so far.

### Other information:


## Jetpack product discussion

* p1HpG7-r96-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start a brand new site
* Install and activate the plugin
* Be redirected to the My Jetpack screen.
* There should be no other sign of the plugin being installed
